### DISCUSSION TITLE
Can't reset creds

### DISCUSSION THREAD

**Discussion by RREXXY** [LizardByte/community#237] [Sunshine] [answered]
From my understanding I can reset my localhost creds with the command (see below) but in the console when I run the exe it doesn't let me type. It's that I don't remember my creds but I probably made a typo when setting it up so I have no way of logging in to the web client currently.

sunshine --creds <new username> <new password>

**Comment by ReenigneArcher** [comment, marked as answer]
The command is `sunshine --creds <new username> <new password>`

See docs: https://docs.lizardbyte.dev/projects/sunshine/en/latest/troubleshooting/general.html#forgotten-credentials
